<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QMainWindow" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Wisard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Templates</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="startDateTimeLabel">
           <property name="text">
            <string>Start Date Time:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="startDateTimeDateTimeEdit">
           <property name="displayFormat">
            <string>dd.MM.yyyy hh:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="endDateTimeLabel">
           <property name="text">
            <string>End Date Time:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="endDateTimeDateTimeEdit">
           <property name="displayFormat">
            <string>dd.MM.yyyy hh:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>554</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="reportTableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>&amp;Report</string>
    </property>
    <addaction name="actionInsertColumn"/>
    <addaction name="actionInsertRow"/>
    <addaction name="actionRemoveRow"/>
    <addaction name="actionRemoveColumn"/>
    <addaction name="actionChoiseFilter"/>
    <addaction name="separator"/>
    <addaction name="actionCreate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QToolBar" name="reportToolBar">
   <property name="windowTitle">
    <string>Report</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertRow"/>
   <addaction name="actionInsertColumn"/>
   <addaction name="actionRemoveRow"/>
   <addaction name="actionRemoveColumn"/>
   <addaction name="actionChoiseFilter"/>
   <addaction name="separator"/>
   <addaction name="actionCreate"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder-horizontal-open.png</normaloff>:/icons/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk-black.png</normaloff>:/icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/report-excel.png</normaloff>:/icons/report-excel.png</iconset>
   </property>
   <property name="text">
    <string>Create Report</string>
   </property>
  </action>
  <action name="actionInsertRow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/table-insert-row.png</normaloff>:/icons/table-insert-row.png</iconset>
   </property>
   <property name="text">
    <string>Insert Row</string>
   </property>
   <property name="toolTip">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="actionInsertColumn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/table-insert-column.png</normaloff>:/icons/table-insert-column.png</iconset>
   </property>
   <property name="text">
    <string>Insert Column</string>
   </property>
   <property name="toolTip">
    <string>Insert Column</string>
   </property>
  </action>
  <action name="actionRemoveRow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/table-delete-row.png</normaloff>:/icons/table-delete-row.png</iconset>
   </property>
   <property name="text">
    <string>Remove Row</string>
   </property>
  </action>
  <action name="actionRemoveColumn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/table-delete-column.png</normaloff>:/icons/table-delete-column.png</iconset>
   </property>
   <property name="text">
    <string>Remove Column</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door-open.png</normaloff>:/icons/door-open.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionChoiseFilter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/script-import.png</normaloff>:/icons/script-import.png</iconset>
   </property>
   <property name="text">
    <string>Choise Filter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>ReportDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
